config/maps.txt

```diff
@@ -17,6 +17,9 @@ map yogstation
 	votable
 endmap
 
+map asteroidstation
+endmap
```

Comment:
```suggestion
	maxplayers 40
endmap
```

Reply:
I'll probably want this at minplayers 35 once testing is done, but for now using it for low-midpop makes sense